.github/workflows/ci_cd.yml

```diff
@@ -213,32 +210,28 @@ jobs:
         if: steps.ui-files-changed.outputs.any_modified != 'true'
         run: echo "value=true" >> $GITHUB_OUTPUT
 
-      - name: Make cache dirs and set file permissions
-        run: |
-          chmod +x gradlew
-          mkdir -p ~/manager-tags-old
-          mkdir -p ~/manager-tags-new
-
       - name: Check if main repo
         id: is_main_repo
         run: |
-          if [ $(grep -E '.+/openremote$' <<< "$GITHUB_REPOSITORY") ]; then
+          if [ -f manager/src/main/java/org/openremote/manager/Main.java ]; then
             echo "value=true" >> $GITHUB_OUTPUT
           else
             if [ -e 'openremote' ]; then
-              echo "::error::Custom project should not have an 'openremote' file or directory!"
+              echo "::error::openremote submodule no longer supported, see custom project template!"
               exit 1
             fi
 
             echo "value=false" >> $GITHUB_OUTPUT
             echo "repository=$(sed -E 's#(.+)/.+#\1/openremote#' <<< $GITHUB_REPOSITORY)" >> $GITHUB_OUTPUT
 
             version=$(cat gradle.properties | grep openremoteVersion | sed -E 's#.+=(.+)#\1#' | xargs)
-            if [ $(grep -E '.+SNAPSHOT$' <<< "$version") ]; then
-              echo "ref=master" >> $GITHUB_OUTPUT
-            else
-              echo "ref=$version" >> $GITHUB_OUTPUT
+
+            if [ -z "$version" ]; then
+              echo "openremoteVersion must be set in gradle.properties"
+              exit 1
             fi
+
+            echo "ref=$version" >> $GITHUB_OUTPUT
```

Comment:
The master value was used so you can more easily use/test snapshot versions in a custom project. That's no longer possible with these changes.

Reply:
We don't build snapshot versions of docker containers